src/pom.xml

```diff
@@ -1012,7 +1012,7 @@
    <dependency>
     <groupId>org.freemarker</groupId>
     <artifactId>freemarker</artifactId>
-    <version>2.3.18</version>
+    <version>2.3.25-incubating</version>
```

Comment:
Yeah, looked stranged to me at first, but nothing to be worried about: http://freemarker.org/freemarkerdownload.html#downloadApacheDisclaimer